### PR TITLE
updated executive list, added new members

### PR DESCRIPTION
**Pull request by lwjohnst86**
@aays @SaraMahallati @nsahin Let me know if this is correct. Otherwise I will merge in.

@mbonsma @linamnt Do you want me to change your positions as well?

@lcoome Since @mbonsma is now the Special Events Coordinator, what is your position? I can't recall what we discussed.

@joelostblom Do you have a new title or you share with @aays?

### CODE REVIEW

**Comment by joelostblom** [comment]
Maybe put me as technical admin as well (we need a fancier name for it to embellish our resumes...)?  And can you change "IBBME" to "Biomedical Engineering"?

**Comment by QuLogic** [comment]
Do we need to wait for elections to make this official?

**Comment by lwjohnst86** [comment]
@QuLogic good question. At least for my position, you don't ... since I'm not there! :P We can put this PR on hold if need be until then. And @joelostblom what about Technical Specialist or Technical Support Specialist or Webmaster or Web Administrator or Technical Operations Officer or Support Specialist...? Thoughts @QuLogic since you also share the title.

**Comment by linamnt** [comment]
Now elections are over, we can also add @mbonsma for Special Events Coordinator, @lcoome Marketing and Social Media Coordinator? @joelostblom any final requests for your role title?

**Comment by lwjohnst86** [comment]
So all are as included in the admin file? @linamnt as president, etc? @QuLogic Still technical admin?

File: _data/admin.yml [hunk 1]
```diff
@@ -1,23 +1,33 @@
...
+  degree: MSc, Biology
+  github: aays
   interests:
     - R
+    - Python
+
+- name: Sara Mahallati
+  position: Admin
+  degree: PhD, Biomedical Engineering
+  github: SaraMahallati
```
Comment (QuLogic) [inline _data/admin.yml]: @SaraMati

**Review by QuLogic — CHANGES_REQUESTED** [review]

**Pull request merged**